sympy/ntheory/digits.py

```diff
@@ -17,12 +17,38 @@ def digits(n, b=10):
     >>> from sympy.ntheory.digits import digits
     >>> digits(35)
     [10, 3, 5]
-    >>> digits(27, 2)
+    >>> digits(27, 2, None)
     [2, 1, 1, 0, 1, 1]
-    >>> digits(65536, 256)
+    >>> digits(65536, 256, None)
     [256, 1, 0, 0]
```

Comment:
IMO, you should not use `None`, instead just use `digits(65536, 256)`

Reply:
Anything else or may I make a final commit?

Reply:
Okay, making the final changes.